src/app/features/swap/use-swap-feature.ts

```diff
@@ -0,0 +1,5 @@
+export const useSwapFeature = () => {
+  return {
+    swapIsEnabled: !!process.env.SWAP_ENABLED
+  };
+}
```

Comment:
Please favour function declarations

```suggestion
export function useSwapFeature() {
  return {
    swapIsEnabled: !!process.env.SWAP_ENABLED
  };
}
```